ORCID.js

```diff
@@ -0,0 +1,99 @@
+{
+	"translatorID": "e83248bb-caa4-4dd2-a470-11f4cd164083",
+	"label": "ORCID",
+	"creator": "Philipp Zumstein",
+	"target": "https?://orcid\\.org/",
+	"minVersion": "3.0",
```

Comment:
That looks like something on their end?
`curl -LH "Accept: application/vnd.citationstyles.csl+json" https://pub.orcid.org/v2.0_rc2/0000-0003-1613-5981/work/4718317` fails with a cryptic 500 error for me, whilc

`curl -LH "Accept: application/xml" https://pub.orcid.org/v2.0_rc2/0000-0003-1613-5981/work/4718317`

and
`curl -LH "Accept: application/xml" https://pub.orcid.org/v2.0_rc2/0000-0003-1613-5981/work/4718317`

fail. @TomDemeranville might you be able to help her (or report this to the right folks?)